ios/FluentUI/Vnext/List/List.swift

```diff
@@ -200,6 +211,15 @@ class MSFListStateImpl: NSObject, ObservableObject, MSFListState {
             }
         }
     }
+
+    @Published var selectionStyle: MSFListSelectionStyle = .trailingCheckmark {
```

Comment:
Does this property and the `isSelectable` above need to be @Published?
It seems to me the only one that needs the property wrapper is the one in the MSFListCellStateImpl because it's directly used to draw/layout the cell View.